### ISSUE TITLE
Closure of VP Github

### ISSUE THREAD

**Issue by vpltd-plesurf** [virtual-programming/overlord-linux#28]
Hi,

We have decided to close VP GitHub in the next seven days.
Our tech team are best able to support users via the company support ticket system.

www.vpltd.com/supportdesk/

If you have an issue that has not been resolved in this area please raise a ticket.

Thank you.

**Comment by eddie-dunn** [comment]
That is unfortunate.

What is the status for solving #7? The bug makes the game almost unplayable, unfortunately. I'd rather not muck around with polling rates for my mouse, since it works fine for all other games.

**Comment by vpltd-kgalaj** [comment]
Still the same, we weren't able to make it appear on any of our setups, so we're no closer to figuring out why it's happening for some people than we were before.

**Comment by ysblokje** [comment]
@vpltd-kgalaj  you might want to talk to uber entertainment (from PA) they had the same issue  with their game and managed to fix it ;)

**Comment by vpltd-kgalaj** [comment]
@ysblokje Judging by contents of this thread ( https://forums.uberent.com/threads/returning-player-severely-low-fps-on-mouse-movement.69196/ ), the issue was on Windows, for most people graphics card driver update fixed it, one of the gamers fixed it for himself, it involved low-level changes in Windows-specific code, and I couldn't find any indication that the issue was ultimately fixed for everyone, just that one official response stated that an engineer will look at it. It doesn't look like it has anything to do with the issue here.

**Comment by ysblokje** [comment]
@vpltd-kgalaj, we're not on windows. I will post the linux related links I got from sigz for reference. 
[](http://steamcommunity.com/app/233250/discussions/2/537402115082303025/)
[](http://steamcommunity.com/app/233250/discussions/2/598198356196274119/)
Now, I wish VP all the best but I hope you guys/girls really up your support "game" in the future.

**Comment by eddie-dunn** [comment]
I didn't mean to start another discussion on issue #7. I merely asked because I wanted to know if I should report the problem in your [new "issue tracker"](http://www.vpltd.com/support/). Unfortunately, it doesn't seem to be public, so there's no way for me to know if it has been reported there, and whether it's being worked on or not.

But I take it you have given up on a fix, since you can't reproduce it? If not, please don't hesitate to get in touch with me, if there's any way I can help you resolve the problem.

**Comment by vpltd-kgalaj** [comment]
Yes, please do report it on the support desk. We maintain a list of issues to fix, and we do work on them when we can.

I am not privy to exact inner workings of Virtual Programming, as I only write code and provide technical support, so I can only speak from my own knowledge here, but from what I've been told, due to the need for the company to remain self-supporting and/or profitable, the tasks that bring in profits are prioritised over the tasks that don't. For example, I've heard that some companies contract the port at Virtual Programming, but don't contract its post-release maintenance. Or, in case of some games, the revenue brought in from various target platforms (like Mac OS X, Linux, XBox, PS) is measured and platforms that don't bring in much revenue aren't at high priority for maintenance. Ultimately, I only know that I fix the issues when I'm specifically tasked to, or when the issues don't consume significant amounts of time or new hardware setups to reproduce and I can fix them as part of general technical support. I don't know the status of Overlord is, but you can be sure that we don't like leaving bugs in games at all, and when we can, we fix them.

I understand that the support desk and its need for everyone to open a separate ticket also serves as the measurement of how widespread the issue is, and how important is it. If it's not inconvenient, people don't bother to open tickets. If it's not widespread, not many people open tickets either. The more separate and independent tickets are opened about it by different people who really have it, the more likely it is to be given to technical team to address sooner.  I've been on a couple of bugs hunts in the past, on which I spent days only to find out that the issue was one failing piece of hardware on one specific machine, so I can understand that approach. With GitHub, hundreds of people may have the issue, but 90% of them would find this place, and leave no comment, expecting it to not matter, and 10% would write a ticket, so the issue might appear 10 times less important for consideration than it really is.
